test/components/previews/selection_preview.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class SelectionPreview < Lookbook::Preview
+  # The selection controller stores the values of checkboxes in local storage. The local storage key is the url.
+  # When a user toggles a checkbox, the value is updated in local storage.
+  def default; end
+
+  # The selection controller stores the values of checkboxes in local storage. The local storage key is the url.
```

Comment:
**praise (non-blocking):** I like the way these add comments into the note section of the Lookbook preview.  Nice!

Reply:
Thanks.